controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -557,6 +557,14 @@ void cli_bml::setFunctionsMapAndArray()
         " is-single-scan - 0 for continuous-scan results (default), 1 for single scan.",
         static_cast<pFunction>(&cli_bml::get_dcs_scan_results_caller), 2, 3, STRING_ARG, INT_ARG,
         INT_ARG);
+    insertCommandToMap("bml_client_get_client_list", "", "Get client list.",
+                       static_cast<pFunction>(&cli_bml::client_get_client_list_caller), 0, 0);
+    insertCommandToMap("bml_client_set_client", "<sta_mac>", "Set client with the given STA MAC.",
```

Comment:
if this API only adds client mac to the DB with default values then where are the rest of APIs that configure rest of client's parameters ?

Reply:
Extended the CLI usage, and now the client needs to add the client parameters